core/test/truffle-fixture.js

```diff
@@ -0,0 +1,36 @@
+const deployFinder = require("../migrations/2_deploy_finder.js");
+const deployTimer = require("../migrations/3_deploy_timer.js");
+const deployVotingToken = require("../migrations/4_deploy_voting_token.js");
+const deployVoting = require("../migrations/5_deploy_voting.js");
+const deployRegistry = require("../migrations/6_deploy_registry.js");
+const deployFinancialContractsAdmin = require("../migrations/7_deploy_financial_contracts_admin.js");
+const deployStore = require("../migrations/8_deploy_store.js");
+const deployGovernor = require("../migrations/9_deploy_governor.js");
+const deployDesignatedVotingFactory = require("../migrations/10_deploy_designated_voting_factory.js");
+const deploySupportIdentifiers = require("../migrations/11_support_identifiers.js");
+const deployTestnetToken = require("../migrations/12_deploy_testnet_token.js");
+const deployTokenfactory = require("../migrations/13_deploy_tokenfactory.js");
+const deployExpiringMultiPartyCreator = require("../migrations/14_deploy_expiring_multi_party_creator.js");
+const deployLocalWeth = require("../migrations/15_deploy_local_weth.js");
+
+const { getKeysForNetwork } = require("../../common/MigrationUtils.js");
+
+module.exports = async ({ network, web3 }) => {
```

Comment:
The network name is likely what's causing the tests to dump a file of addresses. Do you know what network name buidler is using? The networks are set up in a pretty precise way, so we need to make sure the configuration matches across toolsets.

Reply:
The default network name would be `builder-evm`. But its just running it for the `tests`.

I think its dumping out the `1234.json` was because I previously ran `./ci/run_truffle_test.sh`.

How are the networks names being used?

Reply:
If the names are a public network _or_ ci (for testing purposes), the migrations will dump the file to track the deployed addresses outside of the truffle artifacts.

Yeah, you're right -- it only dumped the file because you ran that script. `buidler-evm` shouldn't cause any problems!